utils/src/abi.rs

```diff
@@ -182,14 +182,22 @@ fn expand_output_param_type(
             Ok(format!("{}[{}]", ty, *size))
         }
         ParamType::Tuple(_) => {
-            let ty = if let Some(struct_name) = structs
-                .get_function_output_struct_type(&fun.name, param.internal_type.as_ref().unwrap())
-            {
-                struct_name.to_string()
+            if param.internal_type.as_ref() == None {
+                let mut result = kind.to_string();
+                result.pop();
+                result.remove(0);
```

Comment:
this removes first and list char.
why is this necessary here? what is the first/last char in this context?

Reply:
It was adding parentheses for the struct variables so removed them but let me check again if it's really necessary, I tested generated interface in our production btw and it works

Reply:
I think we can just `trim_start_matches`/`trim_end_matches` then?

Reply:
Yeah changed to use it thanks for recommendation